#Main Dimensions of Stator Core

\begin{equation}
  D_{is} = \sqrt[3]{\frac{2p*p_1*S_{gap}}{pi*\lambda*f_1*C_0}}
\end{equation}

In [5]:
P_n = 150 
V_1 = 1000
n_1 = 1800 
p   = 2
p_1 = 2
f_1 = 50
cos_theta = 0.85
K_E = 0.98 - 0.005*p_1
eff = 0.9
q=3


\begin{equation}
  S_{gap} = \frac{K_E*P_n}{\eta*\cos(\theta_n)}
\end{equation}

In [28]:
print K_E

0.97


In [11]:
S_gap = ((K_E*150*1000)/(eff*cos_theta))
print S_gap

190196.078431


###for a 4 pole machine,
\begin{equation}
\lambda = 1.1
\end{equation}

In [9]:
import math
D_is = math.pow((2*p*p_1*S_gap)/(3.14*1.1*f_1*265000),1.0/3.0)
print D_is

0.321551923037


\begin{equation}
D_{is} = 0.32 m
\end{equation}

\begin{equation}
\tau = \frac{pi*D_{is}}{2*p}
\\\tau_{s} = \frac{\tau}{3*q}
\end{equation}

In [12]:
pole_pitch = 3.14*D_is/(2*p)
print pole_pitch

0.252418259584


\begin{equation}
\tau = 0.25 m
\end{equation}

In [13]:
sl_per_pole= pole_pitch/(3*q)
print sl_per_pole

0.0280464732871


####again for a 4 pole machine;
\begin{equation}
\frac{D_{is}}{D_{out}} = K_D = 0.62
\end{equation}

In [14]:
D_out = D_is/0.62
print D_out

0.51863213393



\begin{equation}
D_{out} = 0.52 m
\end{equation}

\begin{equation}
g = (0.1 + 0.012*\sqrt[3]{P_n}*10^{-3}
\\for \quad 2p>2
\end{equation}

In [15]:
import math
g = (0.1+0.012*math.pow(P_n*10^3,1.0/3.0))/(10^3)
print g

0.0263841361402


\begin{equation}
g = 0.26 mm
\end{equation}

#The Stator Winding 

\begin{equation}
N_s = 2*p*q*m = 2*2*3*3 = 36
\end{equation}

\begin{equation}
\alpha_{ec} = \frac{2\pi*p}{N_s} = \frac{\pi}{9}
\end{equation}

\begin{equation}
K_{ql} = \frac{\sin(\frac{\pi}{6})}{q*\sin(\frac{\pi}{6q})}
\end{equation}

In [16]:
import math
K_ql = math.sin(3.14/6)/(3*math.sin(3.14/18))
print K_ql

0.959835420315


\begin{equation}
for\quad chording : \frac{y}{\tau} = \frac{7}{9}\quad is \quad selected
\end{equation}

\begin{equation}
so\quad the\quad electrical\quad angle\quad is:\quad \frac{2*\pi*p}{N_s}=\quad \frac{\pi}{9}
\end{equation}

\begin{equation}
chording\quad factor\quad is:\quad \sin(\frac{pi*y}{2*\tau})
\end{equation}

In [17]:
import math
K_yl = math.sin((3.14*7)/(2*9))
print K_yl

0.939480605157


In [18]:
import math
Kw1= K_yl*K_ql
print Kw1

0.901746761529


####following values are assumed from now on:


In [89]:
B_g = 0.7 
alpha_i = 0.65
L=1.1*3.14*D_is/(2*p)
rated_current = 265000/(1.1*alpha_i*Kw1*3.14*B_g*p)
print rated_current/1000


93.4969743854


In [66]:
pole_flux=alpha_i*pole_pitch*L*B_g
print pole_flux

0.0318893463746


####number of turns per phase W1:
\begin{equation}
\frac{K_E*V_1}{4*K_f*K_{W1}*f*\Phi}
\end{equation}

In [67]:
import math
W_1 = (K_E*V_1)/(4*1.1*Kw1*50*pole_flux*math.sqrt(3))
print W_1 

88.5234267262


####number of conductors ns:
\begin{equation}
ns:\quad \frac{W_1}{p*q}
\end{equation}
#### (current paths in parallel:1)

In [68]:
ns = W_1/(p*q)
print ns

14.7539044544


In [70]:
ns = 14

In [71]:
import math
rated_current= 150000/(eff*cos_theta*math.sqrt(3)*V_1)
print rated_current


113.205935135


In [72]:
J_co=6.3

####magnetic wire cross section:
\begin{equation}
A_{co}=\quad \frac{I_{rated}}{J_{cos}}
\end{equation}

In [73]:
A_co=rated_current/J_co
print A_co

17.9691960532


\begin{equation}
A_{co}=conductor\quad width*conductor\quad height
\end{equation}
####a rectengular cross section conductor will be used,so standart value of 10 mm is taken here for slot width

In [83]:
slot_width = 10
conductor_insulation = 0.4
slot_insulation = 4
conductor_width = slot_width - conductor_insulation - slot_insulation
print conductor_width

5.6


#### conductor height = A_co/conductor width = 18/5.6 = 3.21mm

In [85]:
radial_cond_insulation = conductor_insulation*ns
radial_slot_insulation = slot_insulation*2
interlayer_insulation = 2
wedge = 4
radial_insulation = radial_cond_insulation + radial_slot_insulation+interlayer_insulation+wedge
print radial_insulation

19.6


####slot height:
\begin{equation}
h_s = h_{ins} + n_s*b_c
\end{equation}

In [86]:
h_s = radial_insulation + ns*3.21
print h_s

64.54


####back iron radial thickness:
\begin{equation}
h_{cs} = \frac{D_{out}-D_{is}}{2}-h_s
\end{equation}

In [87]:
h_cs= ((D_out-D_is)*1000)/2-h_s
print h_cs

34.0001054468


####so all in all;
\begin{equation}
h_s = 64.54 mm
\\h_{cs} = 34 mm
\end{equation}
####thus back iron flux density is:
\begin{equation}
B_{cs} = \frac{\Phi}{2*l_i*h_cs}
\end{equation}
####where l_i is stack length, and calculated as 0.277mm

In [91]:
B_cs = pole_flux/(2*L*0.001*h_cs)
print B_cs

1.68896988114


#### 1.6 T is an acceptable value, so no need to change the calculations for stator diameters